packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_withdraw_from_vault.test.ts

```diff
@@ -578,8 +578,8 @@ describe('withdrawFromVault with 50% DUSD or DFI collaterals', () => {
     }, bobVaultScript)
     let outs = await sendTransaction(bob.container, txn)
     expect(outs[0].value).toStrictEqual(0)
-    expect(outs[1].value).toBeLessThan(20)
-    expect(outs[1].value).toBeGreaterThan(19.999)
+    expect(outs[1].value).toBeLessThan(10)
```

Comment:
I don't quite understand why the use of this specific value here(why use 10 instead of 20, can't really see why in this file). Can you give me some pointers as to what it is and maybe add a comment to add context?

Reply:
inside `beforeEach` you fund 2*10 utxos for this test
```
// Fund 10 DFI UTXO to providers.getAddress() for funds
    await fundEllipticPair(alice.container, providers.ellipticPair, 10)
    await fundEllipticPair(bob.container, providers.ellipticPair, 10)
```
earlier, the mock provider considered all utxos available as inputs for the tx. hence 20.
But now it will consider utxos to suffice `minimumSumAmount`. Hence it stops at 10.
check here for the code -> https://github.com/JellyfishSDK/jellyfish/pull/1362/files#diff-181aaecd523f5e53cf95cc219b688705d130741a113f473ab0fdec59697b5584R46-R68